spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -459,11 +475,19 @@
           )
           expect(job_analytics).to have_logged_event(
             'GetUspsProofingResultsJob: Success or failure email initiated',
-            delay_time_seconds: 3600,
             email_type: 'Success',
             service_provider: anything,
             timestamp: anything,
             user_id: anything,
+            wait_until: nil,
```

Comment:
Are we properly verifying that this field is logged when it does get populated?

Reply:
No, I'll add some additional checks for this field

Reply:
Added in a52f5e5f6633f41890c31baec9b17cc2e591900e